pkgs/applications/terminal-emulators/rio/default.nix

```diff
@@ -55,16 +55,18 @@ let
 in
 rustPlatform.buildRustPackage rec {
   pname = "rio";
-  version = "0.1.8";
+  version = "0.1.10";
 
   src = fetchFromGitHub {
     owner = "raphamorim";
     repo = "rio";
     rev = "refs/tags/v${version}";
-    hash = "sha256-NLvCLyIts3NoSqbR9PIhXj3GCeURrBXXv/zVgZecfNQ=";
+    hash = "sha256-S42is3wqjBvYgysQ6yDUAn7ZEy9xJBmQD/emYAQfCkw=";
   };
 
-  cargoHash = "sha256-TvNmTI3At2ezsdEOd5mAc6uUQXgGP47d0AMVVUUJr/A=";
+  buildType = "debug";
```

Comment:
Why?

Reply:
Oh whoops, I'm adding this only to build it locally. I forgot to remove it afterwards xD Good catch!